File: python/create_trunk.py
#!/usr/bin/env python3

from _client import post
import os

# Create trunk with environment variables
trunk_data = {
    'trunk_name': os.getenv('TRUNK_NAME', 'my_ai_trunk'),
    'nso_code': os.getenv('NSO_CODE', 'ANY-ANY'),
    'domain_name': f"{os.environ['EXO_ACCOUNT_SID']}.pstn.exotel.com"
}

print("Creating trunk...")
result = post('/trunks', trunk_data)
print("Trunk created successfully!") 

File: php/create_trunk.php
#!/usr/bin/env php
<?php

require '_client.php';

// Create trunk with environment variables
$trunk_data = [
    'trunk_name' => getenv('TRUNK_NAME') ?: 'my_ai_trunk',
    'nso_code' => getenv('NSO_CODE') ?: 'ANY-ANY',
    'domain_name' => getenv('EXO_ACCOUNT_SID') . '.pstn.exotel.com'
];

echo "Creating trunk...\n";
$result = exo_post('/trunks', $trunk_data);
echo "Trunk created successfully!\n"; 